xbmc/windowing/egl/EGLNativeTypeAmlogic.cpp

```diff
@@ -220,6 +224,29 @@ void CEGLNativeTypeAmlogic::SetupVideoScaling(const char *mode)
   SysfsUtils::SetInt("/sys/class/graphics/fb0/blank", 0);
 }
 
+void CEGLNativeTypeAmlogic::DealWithScale(const RESOLUTION_INFO &res)
```

Comment:
I know it may sound picky, but "DealWith" remembers me on drugs.
If everybody is fine with this I don't care, for me a better name would be: HandleScale because its mor in common with kodi naming conventions.